internal/api/approval/approval.go

```diff
@@ -400,3 +425,21 @@ func filterAllowedAndIgnoreReviewers(members []*github.User, commits []*github.R
 
 	return allowed, ignored
 }
+
+func findCoAuthors(msg string) []string {
+	pattern := "Co-authored-by: .+? <([\\w\\+-]+)@users.noreply.github.com>"
+	r := regexp.MustCompile(pattern)
```

Comment:
nit, but it is pointless to compile pattern every time `findCoAuthors` is invoked